expression/builtin_string_vec.go

```diff
@@ -62,6 +62,31 @@ func (b *builtinRepeatSig) vecEvalString(input *chunk.Chunk, result *chunk.Colum
 	return nil
 }
 
+func (b *builtinStringIsNullSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.AppendInt64(1)
+		} else {
+			result.AppendInt64(0)
+		}
+	}
+	return nil
+}
+
 func (b *builtinRepeatSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinStringIsNullSig) vectorized() bool {
+	return true
+}
```

Comment:
I think it's better to put each evaluation function and its vectorized function together. Not all of the vectorized functions together. What's your opinion? @qw4990

Reply:
I agree with you.

Reply:
ok